src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Rejection.scala

```diff
@@ -47,6 +48,15 @@ object Rejection {
   final case class UpdateSchemaTypes(ref: Ref)
       extends Rejection(s"Resource '${ref.show}' cannot change it's schema type.")
 
+  /**
+    * Signals an attempt to perform a request with an invalid payload.
+    *
+    * @param ref a reference to the resource
+    * @param json the incorrect json payload
+    */
+  final case class InvalidPayload(ref: Ref, json: Json)
```

Comment:
I'd recommend using a `reason: String` argument instead of the `json` argument to explain why the payload is invalid.